includes/services/extended/instagram-basic-display.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/**
+ * Instagram service definition for Keyring.
+ * http://instagram.com/developer/
+ */
+
+class Keyring_Service_Instagram_Basic_Display extends Keyring_Service_OAuth2 {
+	const NAME  = 'instagram_basic_display';
+	const LABEL = 'Instagram Basic Display';
```

Comment:
Maybe just go with "Instagram Basic" which is a little friendlier here. This label gets used in UI etc.

Reply:
fixed